.gitmodules

```diff
@@ -58,3 +58,6 @@
 [submodule "src/rapidjson"]
 	path = src/rapidjson
 	url = https://github.com/ceph/rapidjson
+[submodule "src/nvml"]
+	path = src/nvml
+	url = https://github.com/ceph/nvml.git
```

Comment:
Yes, now it's not exist.  I don't know how to fork a submodule . Maybe it need Sage to do this .